README.md

```diff
@@ -1,17 +1,100 @@
 # Conda Recipes
 
-Example recipes for the new conda build system.  Use
+Community-maintained repository of conda package build recipes
+for 592+ conda packages.
 
-    conda build <recipe directory>
+* Package maintainers SHOULD upgrade packages
+  when appropriate.
 
-You have to use conda >= 1.7
-[conda](https://github.com/continuumio/conda).
+* Conda recipes MAY run other scripts:
 
-See http://docs.continuum.io/conda/build.html for information on how to make a recipe,
-or just look at the examples here.
```

Comment:
Still there in line 36: https://github.com/conda/conda-recipes/pull/290/files#diff-04c6e90faac2675aa89e2176d2eec7d8R36 :0)